scripts/lint-style.py

```diff
@@ -133,7 +133,7 @@ def small_alpha_vrachy_check(lines, path):
 def instance_check(lines, path):
     errors = []
     for line_nr, line in skip_string(skip_comments(enumerate(lines, 1))):
-        if re.search(r'\b(_inst_)\d+', line) is not None:
+        if re.search(r'\b(?<!`)(_inst_)\d+', line) is not None:
```

Comment:
Yes, exactly. I'm generating a lemma statement and need to name the instance arguments; so I'm calling them `_inst_1`. The linter gets annoyed at me because it thinks I'm writing a proof and referring to `_inst_1`. This seems like a reasonable way to silence it.